app/commands/notification/create.rb

```diff
@@ -6,10 +6,13 @@ class Create
 
     def call
       klass = "notifications/#{type}_notification".camelize.constantize
-      klass.create!(
-        user: user,
-        params: params
-      )
+
+      klass.create!(user: user, params: params).tap do
```

Comment:
Just a note: Pending the decision on how whether we'd rather standardise on using the success handler, we might want to refactor this tap to be `on_success` instead.

Reply:
How do we do this? Re-wrap this into another command?

Reply:
Ah - I actually misread this slightly. Let's discuss this one on the call. I'll think about whether I want to use those callbacks or not today.

Reply:
I'll resolve this now. I've added this to our tracking issue.